client/src/utils.rs

```diff
@@ -0,0 +1,16 @@
+// List of Succinct Gateway addresses for chains.
+const GATEWAY_ADDRESSES: [(u32, &str); 6] = [
```

Comment:
lol why not make this a `HashMap`?

Reply:
Not static, thought it was cleaner this way